### PR TITLE
Fix replaced nan with zero

### PR DESCRIPTION
**Pull request by abhishekkhanduri**
#2037

### Description
Replaced NaN with zero when gross salary is not present.
### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

**Comment by abhishekkhanduri** [comment]
image
Before Fix

**Comment by abhishekkhanduri** [comment]
image
After fix

File: Modules/Salary/Resources/assets/js/components/SalaryBreakdown.vue [hunk 1]
```diff
@@ -254,6 +254,9 @@ export default {
 			return Math.min(Math.ceil(multiplier * percentage / 100) , Math.ceil(this.salaryConfigs.edli_charges_limit.fixed_amount * percentage / 100));
 		},
 		ctc() {
+			if (this.grossSalary === "") {
```
Comment (rathorevaibhav) [inline Modules/Salary/Resources/assets/js/components/SalaryBreakdown.vue:257]: is `NaN` only showing at one place?

**Review by rathorevaibhav — APPROVED** [review]

File: Modules/Salary/Resources/assets/js/components/SalaryBreakdown.vue (hunk 1 above)
Comment (abhishekkhanduri) [reply Modules/Salary/Resources/assets/js/components/SalaryBreakdown.vue:257]: It was showing NaN at three places, CTC, CTC Annual, CTC Aggreed and as I was using CTC in calculating other two values. Resolving CTC value worked for me in resolving other values as well.

**Pull request merged**